cargo-typify/src/main.rs

```diff
@@ -0,0 +1,66 @@
+use std::path::PathBuf;
+
+use clap::Parser;
+
+use color_eyre::eyre::{Context, Result};
+use schemars::schema::Schema;
+use typify::{TypeSpace, TypeSpaceSettings};
+
+/// A CLI for the `typify` crate. Which allows conversion of JSON Schema files to Rust types
+#[derive(Parser, Debug)]
+#[command(author, version, about)]
+struct Args {
```

Comment:
Good start; I'd like to be able to specify all the `TypeSpaceSettings`... somehow...

I'd especially like to support the `struct_builder` option.